apollo/integrations/db/base_db_proxy_client.py

```diff
@@ -23,8 +23,13 @@ def __init__(self):
 
     # On delete make sure we close the connection
     def __del__(self) -> None:
+        self.close()
+
+    def close(self):
         if self._connection:
+            logger.info("Closing DB Proxy connection")
```

Comment:
Would be useful if we can also log the client type (e.g., redshift etc.)

Reply:
good point, done